jobs/video_jobs/maya_submit.sh.erb

```diff
@@ -19,7 +19,7 @@ CMD="singularity exec $MAYA_IMG Render"
 RENDERER="<%= attributes['renderer'] %>"
 START_FRAME="<%= start_frame %>"
 END_FRAME="<%= end_frame %>"
-EXTRA_ARGS="<%= attributes['extra_args'] %>"
+EXTRA_ARGS="<%= attributes['extra'] %>"
```

Comment:
use the getter "extra" not the attributes hash; i did like extra_args better than extra (more expressive) but can appreciate if the underscore caused any problems for you